cardy/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "next-cardigan",
```

Comment:
Let's just leave it as cardigan... no need to be next as we should deprecate the last one pretty quick.